src/lib/auth.ts

```diff
@@ -77,6 +78,7 @@ export const ROLES: {
 		'resin.actor.delete?id eq @__ACTOR_ID',
 		'resin.api_key.read?is_of__actor eq @__ACTOR_ID',
 		'resin.application.all',
+		'resin.device_type.all',
```

Comment:
As things stand, device types are currently controlled by balenaCloud (given we're hardcoding our production S3 bucket into openBalena), therefore I think read-only permissions are better suited for this type of thing right now, if only to ensure an installation doesn't break by accident.